sdk/storage/storage-common/src/index-browser.mts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export * from "./BufferScheduler.js";
+export { getCachedDefaultHttpClient } from "./cache.js";
+
+export * from "./StorageBrowserPolicyFactory.js";
+export * from "./credentials/AnonymousCredential.js";
+export * from "./credentials/Credential.js";
```

Comment:
The `StorageSharedKeyCredential` also needs to be exported here

Reply:
IIRC it's not safe to expose `StorageSharedKeyCredential` in browsers as secrets can be easily accessed.